pnpm-lock.yaml

```diff
@@ -11947,8 +11947,9 @@ packages:
         optional: true
     dependencies:
       whatwg-url: 5.0.0
+    patched: true
 
-  /node-fetch/2.6.7_3wcp6bao3dfksocwsfev5pt7km:
+  /node-fetch/2.6.7:
```

Comment:
Only used via `"vercel": "28.13.0"` in `prisma/packages/client/tests/e2e/` tests like `nextjs-schema-not-found/16_monorepo-serverComponents-customOutput-reExportIndirect/packages/service/package.json`

Reply:
and actually `checkpoint-client/1.1.23` so I'll fix that